### PR TITLE
critical gov action deployments

### PR DESCRIPTION
**Pull request by DZGoldman**

### CODE REVIEW

**Comment by DZGoldman** [comment]
(verified)

**Comment by DZGoldman** [comment]
okay ready for review; we can remove the .json files I suppose

File: scripts/deployCriticalGovActions.ts [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+
+const deployAll = async () => {
+  const l1Deployer = new Wallet(l1Key, new JsonRpcProvider(l1RPC));
+
+  const hasBal = (await l1Deployer.getBalance()).gt(0);
+  if (!hasBal) {
+    throw new Error("L1 deployer has no balance");
+  }
+  const chainID = await l1Deployer.getChainId();
+  if (chainID === 5 && l2ChainID !== 412163) {
```
Comment (gvladika) [inline scripts/deployCriticalGovActions.ts:78]: also here

**Review by gvladika — APPROVED** [review]
lgtm

**Pull request merged**